From 9b850a655e136863fcbabeb41890a397f8777031 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <josef.schlehofer@nic.cz>
Date: Sun, 8 Dec 2019 10:14:06 +0100
Subject: [PATCH] Revert "suricata-bin: Add two dependencies - libnss and nspr"

Those packages are just available in OpenWrt 19.07 and OpenWrt master.
They are not present in OpenWrt 18.06 and as those are libraries they
can break things in other packages. It would be easier to disable in via
configure paramaters in Suricata, but we will need wait for newer
version.

More details: https://github.com/OISF/suricata/pull/3309

This reverts commit a971adb8ecf0d3b437785c4b709670266ca248cb.
---
 net/suricata-bin/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/net/suricata-bin/Makefile b/net/suricata-bin/Makefile
index f7afa2c37..81acbc5bf 100644
--- a/net/suricata-bin/Makefile
+++ b/net/suricata-bin/Makefile
@@ -9,7 +9,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=suricata-bin
 PKG_VERSION:=4.0.7
-PKG_RELEASE=2
+PKG_RELEASE=1
 
 PKG_SOURCE:=suricata-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://www.openinfosecfoundation.org/download/
@@ -49,7 +49,7 @@ define Package/suricata-bin
   CATEGORY:=Network
   TITLE:=Network IDS, IPS and NSM engine
   URL:=https://suricata-ids.org/
-  DEPENDS:=+libpcre +libyaml +jansson +libmagic +libpthread +librt +libpcap $(LUAJIT_DEP) +libnetfilter-queue +iptables-mod-nfqueue +libnetfilter-log +libnfnetlink +iptables-mod-conntrack-extra +libhtp +libnet-1.2.x +nspr +libnss
+  DEPENDS:=+libpcre +libyaml +jansson +libmagic +libpthread +librt +libpcap $(LUAJIT_DEP) +libnetfilter-queue +iptables-mod-nfqueue +libnetfilter-log +libnfnetlink +iptables-mod-conntrack-extra +libhtp +libnet-1.2.x
 endef
 
 define Package/suricata-bin/install
-- 
2.24.0

